### DISCUSSION TITLE
missing gnumeric on elementary OS 7 Horus / Ubuntu 22.04.2 LTS / Linux 5.19.0-41-generic

### DISCUSSION THREAD

**Discussion by urbi3** [elementary/triage#231] [Q&A] [answered]
hi
I expected Gnumeric on elementary, but can't even find it on the AppCenter.
What is the easiest way to install it?
Thanks for help

**Comment by danirabbit** [comment, marked as answer]
The best way to get Gnumeric is probably to download it from Flathub and install with Sideload: https://flathub.org/apps/org.gnumeric.Gnumeric

**Reply by urbi3** [reply]
really simple and easy to install, I'm amazed! Thanks a lot
